python/examples/handler.py

```diff
@@ -45,10 +39,7 @@ def say_ok(request, response, proxy):
     yield print_arg(request, 1)
     yield print_arg(request, 2)
 
-    response.argstreams = [
-        InMemStream(),
-        InMemStream(),
-        InMemStream("world")]
+    response.set_body(InMemStream("world"))
```

Comment:
`set_body` and `write_body` are too easy to confuse. As a user I'd rather have one method that can take a string *or* a mem stream, that way I don't need to remember specifics like that.